hms-api/package.json

```diff
@@ -16,16 +16,18 @@
     "eslint": "8.19.0",
     "eslint-config-google": "0.14.0",
     "jest": "27.5.1",
+    "serverless-dynamodb-local": "^0.2.40",
```

Comment:
since we build the backend and package everything together as well as using dependabot to keep things up to date we don't need to use `^`.  Instead we can just specify the exact version